base/interactiveutil.jl

```diff
@@ -571,11 +571,10 @@ end
 downloadcmd = nothing
```

Comment:
I think `downloadcmd` can be removed and just be a local variable.
However this is unrelated to the changes in this PR. Let's save this for   a separate PR